sdk/turing/router/router.py

```diff
@@ -75,8 +75,11 @@ def status(self) -> RouterStatus:
         return self._status
 
     @property
-    def config(self) -> 'RouterConfig':
-        return RouterConfig(name=self.name, environment_name=self.environment_name, **self._config)
+    def config(self) -> Optional['RouterConfig']:
+        if self._config is not None:
+            return RouterConfig(name=self.name, environment_name=self.environment_name, **self._config)
+        else:
+            return None
```

Comment:
Added this `if-else` check to handle cases whereby querying the API for a certain router returns an object without a `RouterConfig`. This occurs notably when a new Turing Router is pending (being deployed).